app/install/ubuntu.md

```diff
@@ -27,7 +27,7 @@ Start by downloading the corresponding package for your configuration:
 
     ```bash
     $ sudo apt-get update
-    $ sudo apt-get install netcat lua5.1 openssl libpcre3 dnsmasq
```

Comment:
I don't think Openresty does it. But the Kong debian package does.

Just installed it on Ubuntu 14, while leaving `lua5.1` out. By installing it using
````
sudo dpkg --ignore-depends=lua5.1 -i kong-0.5.4.trusty_all.deb
````
Everything worked out of the box.

Anyway, I would appreciate it if someone else could look into this. I have a Windows background and I'm not familiar enough (yet) with this package stuff. For now it would take too much time for me to be efficient.